### PR TITLE
chore(config): Update field labels for the rest of the sources and transforms fields

### PR DESCRIPTION
**Pull request by maycmlee**
This PR updates the rest of the sources and transforms' field labels that needs a human_name label. Related to https://github.com/vectordotdev/vector/pull/17517.

### CODE REVIEW

File: src/sources/kubernetes_logs/mod.rs [hunk 1]
```diff
@@ -198,6 +200,7 @@ pub struct Config {
     /// in the underlying file server, so setting it too low may introduce
     /// a significant overhead.
     #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
+    #[configurable(metadata(docs::human_name = "Glob Minimum Cooldown"))]
```
Comment (maycmlee) [inline src/sources/kubernetes_logs/mod.rs:203]: Is `Glob` correct here or should it be `Global`?

File: src/sources/file.rs [hunk 2]
```diff
@@ -211,14 +214,15 @@ pub struct FileConfig {
     #[serde(default)]
     pub oldest_first: bool,
 
-    /// Timeout from reaching `EOF` after which the file is removed from the filesystem, unless new data is written in the meantime.
+    /// Timeout, in seconds, from reaching `EOF` after which the file is removed from the filesystem, unless new data is written in the meantime.
```
[the remark this answers is not in the document]
Comment (spencergilbert) [reply src/sources/file.rs:217]: > After reaching EOF, how many seconds to wait before removing the file - unless new data is written

The `and when the file is removed` seems awkward in your suggestion.

Maybe:

> The wait time, in seconds, between reaching an `EOF` and removing the file from the filesystem, unless new data is written in the meantime.

[the remark this answers is not in the document]
Comment (maycmlee) [reply src/sources/file.rs]: Oh, I like your first description. Used that with a minor edit.

File: src/transforms/lua/mod.rs [hunk 3]
```diff
@@ -39,7 +39,7 @@ pub struct LuaConfigV1 {
     config: v1::LuaConfig,
 }
 
-/// Marker type for the version two of the configuration for the `lua` transform.
+/// Marker type for version two of the configuration for the `lua` transform.
```
Comment (spencergilbert) [inline src/transforms/lua/mod.rs:42]: The `enum V1` has the same wording, should it be replaced as well?

File: src/sources/util/http/auth.rs [hunk 4]
```diff
@@ -11,7 +11,7 @@ use warp::http::HeaderMap;
 ))]
 use super::error::ErrorMessage;
 
-/// HTTP Basic authentication configuration.
+/// HTTP basic authentication configuration.
```
Comment (spencergilbert) [inline src/sources/util/http/auth.rs:14]: Should this be capitalized? https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#basic_authentication_scheme

File: src/sources/kubernetes_logs/mod.rs [hunk 5]
```diff
@@ -167,6 +168,7 @@ pub struct Config {
     #[serde(default)]
     #[configurable(metadata(docs::type_unit = "seconds"))]
     #[configurable(metadata(docs::examples = 600))]
+    #[configurable(metadata(docs::human_name = "Ignore Older Files"))]
```
Comment (spencergilbert) [inline src/sources/kubernetes_logs/mod.rs:171]: (Not knowing how this is presented) would it make sense to frame this as `Ignore Files Older Than`?

File: src/line_agg.rs [hunk 6]
```diff
@@ -66,6 +66,7 @@ pub struct Config {
     /// The maximum amount of time to wait for the next additional line, in milliseconds.
     ///
     /// Once this timeout is reached, the buffered message is guaranteed to be flushed, even if incomplete.
+    #[configurable(metadata(docs::human_name = "Timeout"))]
```
Comment (spencergilbert) [inline src/line_agg.rs:69]: Another case where I'd expect it to not need an additional customization...

File: src/transforms/lua/mod.rs (hunk 3 above)
Comment (maycmlee) [reply src/transforms/lua/mod.rs:42]: Yep, updated it.

File: src/sources/kubernetes_logs/mod.rs (hunk 5 above)
Comment (maycmlee) [reply src/sources/kubernetes_logs/mod.rs]: Make sense, updated.

**Review by spencergilbert — APPROVED** [review]
I'm getting pretty good at these, maybe I have a future in technical writing 😉

**Pull request merged**